otter/convergence/steps.py

```diff
@@ -50,6 +50,69 @@ def set_server_name(server_config_args, name_suffix):
     return server_config_args.set_in(('server', 'name'), name)
 
 
+def _try_json_message(maybe_json_error, keys):
```

Comment:
https://github.com/rackerlabs/otter/issues/1159, since this is a slightly bigger change, and this PR has a big enough diff.